programs/install/Install.cpp

```diff
@@ -239,12 +239,12 @@ int mainEntryClickHouseInstall(int argc, char ** argv)
         uint32_t path_length = 0;
         _NSGetExecutablePath(nullptr, &path_length);
         if (path_length <= 1)
-            Exception(ErrorCodes::FILE_DOESNT_EXIST, "Cannot obtain path to the binary");
```

Comment:
This looks like a real fix to me

Reply:
Yeah ... it affects the error path though which might be why nobody noticed yet 😄.